.github/workflows/jpa_integration_ci.yml

```diff
@@ -6,9 +6,26 @@ on:
   push:
     branches:
       - main
+    paths-ignore:
+      - 'LICENSE*'
+      - '.gitignore'
+      - '**.md'
+      - '**.adoc'
+      - '*.txt'
+      - 'build/**'
```

Comment:
The `build` directory contains the `optaplanner-build-parent`; the parent pom nearly every other module inherits from. That's why should cover it by CI.

This applies to every `.yml` file changed in this PR.

Reply:
```suggestion
```

Reply:
@rsynek applied to every `.yml` file